src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -457,6 +468,8 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration)
             // Start page should not show up during test mode.
             this.ShowStartPage = !DynamoModel.IsTestMode;
 
+            this.BrandingResourceProvider = resourceProvider ?? new DefaultBrandingResourceProvider();
+
```

Comment:
Like all other interfaces, this step should be done in `DynamoViewModel Start()` method above.